concepts/05 UI Components/Form/01 Getting Started with Form/40 Organize the Form Items/10 In Columns.md

```diff
@@ -0,0 +1,164 @@
+The Form can organize items into a fixed number of columns or automatically adjust the layout based on the screen width. To set a fixed number of columns, initialize the [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount) property as done in the code below. To create an adaptive layout, configure the [screenByWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#screenByWidth) and [colCountByScreen](/Documentation/ApiReference/UI_Components/dxForm/Configuration/colCountByScreen/) properties. 
```

Comment:
```suggestion
The Form can organize items into a fixed number of columns or automatically adjust the layout based on the screen width. To specify the number of columns, initialize the [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount) property as shown in the code below. To create an adaptive layout, configure the [screenByWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#screenByWidth) and [colCountByScreen](/Documentation/ApiReference/UI_Components/dxForm/Configuration/colCountByScreen/) properties. 
```

Reply:
We have two types of the Form layout: one has the fixed number of columns regardless of the screen width, another one has different number of columns for different devices. How about the following:
'To keep the fixed number of columns, inintialize...'